Modules/ThirdParty/MINC/src/libminc/volume_io/Volumes/input_mgh.c

```diff
@@ -16,6 +16,9 @@
 #endif
 #include "znzlib.h"
 #include "errno.h"
+#ifdef DEBUG
```

Comment:
`_DEBUG`?

Reply:
There was some differences between the files, but I kept the pattern in upstream.